index.js

```diff
@@ -38,6 +38,10 @@ var validate = module.exports = function (name) {
     errors.push('name cannot start with an underscore')
   }
 
+  if (name.length && !name.match(/^[a-z0-9]/)) {
```

Comment:
Should @ be allowed as 1st char as scoped package names needs that ?

Reply:
Added a comment in the issue and tagged Stefan to confirm this

Reply:
Yes, it should be allowed. Fixed it.